infra/terraform/modules/auth0/rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This rule is managed by Terraform, DO NOT EDIT!
+ */
+function (user, context, callback) {
+    var rangeCheck = require('range_check');
+
+    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (Dev)';
```

Comment:
Should interpolate the terraform `env` var into this label - which would mean templating this file (and probably all rules JS): https://www.terraform.io/docs/providers/template/d/file.html

Reply:
As discussed. This whole label should be passed in, really - if this module is to be usable by people other than us. Easiest way is with a rule config key/val pair, accessed in JS with `configuration[KEY]`

Reply:
tbh I wouldn't worry too much about this being reusable in other contexts - the setup here is pretty specific to this product.